app/invocation/invocation_cache_card.tsx

```diff
@@ -53,13 +55,13 @@ export default class CacheCardComponent extends React.Component<Props> {
                         <div>
                           <div className="cache-chart-label">
                             <span className="color-swatch cache-hit-color-swatch"></span>
-                            <span className="cache-stat">{format.formatWithCommas(cacheStat.actionCacheHits)} </span>
-                            hits
+                            <span className="cache-stat">{format.formatWithCommas(cacheStat.actionCacheHits)}</span>
+                            &nbsp;hits
```

Comment:
nit: consider `white-space: nowrap` instead of using non-breaking spaces here

Reply:
I had to add these because adding `white-space: nowrap` killed the spaces (and I borked the spaces before in a previous PR from formatting with prettier), so I felt like adding an explicit &nbsp; would make this less brittle (I generally avoid them, but this felt like a place where they were useful). I tried {" "} and a bunch of incantations, but nothing else worked reliably in this situation with a span next to text on a new line with nowrap enabled.